src/argus/notificationprofile/admin.py

```diff
@@ -38,9 +38,25 @@ class FilterAdmin(admin.ModelAdmin):
     raw_id_fields = ("user",)
 
 
+class HasPhoneNumberFilter(admin.SimpleListFilter):
+    title = "has phone number"
+    parameter_name = "has_phone_number"
+
+    def lookups(self, request, model_admin):
+        return (("yes", "Yes"), ("no", "No"))
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value == "yes":
+            queryset = queryset.exclude(phone_number__isnull=True)
+        elif value == "no":
+            queryset = queryset.filter(phonenumber__isnull=True)
```

Comment:
I think you missed an underscore 😅 
```suggestion
            queryset = queryset.filter(phone_number__isnull=True)
```

<hr />
Also, should we extract the query parameter into a `Q` object, since they're equal in both branches of the `if` statement? 🤔

Reply:
Thx for spotting the typo! As for a Q: YAGNI I think. I usually wait to abstract until I use the same thing in at least *three* places :) If I make a YesNoFilter, this would be a parameter set on the subclass, as well.

Reply:
Changed in new commit, see previous comment.